Sources/RichTextKit/Format/RichTextFormatSheet.swift

```diff
@@ -62,7 +62,8 @@ public struct RichTextFormatSheet: View {
                         RichTextColorPicker(
                             icon: .richTextColorBackground,
                             value: context.binding(for: .background),
-                            quickColors: .quickPickerColors)
+                            quickColors: .quickPickerColors,
+                            type: .background)
```

Comment:
Change these to `.backgroundColor` and add `.foregroundColor` to the other pickers.